dev/3fb84ce0-605c-4dc6-a2c6-a85a1441f5da/device.lp

```diff
@@ -0,0 +1,18 @@
+(librepcb_device 3fb84ce0-605c-4dc6-a2c6-a85a1441f5da
+ (name "DMC3071LVT")
+ (description "Diodes Incorporated DMC3071LVT Dual MOSFET N/P-Channel 30V.\n\nDatasheet:\n- h\n- t\n- t\n- p\n- s\n- :\n- /\n- /\n- w\n- w\n- w\n- .\n- d\n- i\n- o\n- d\n- e\n- s\n- .\n- c\n- o\n- m\n- /\n- a\n- s\n- s\n- e\n- t\n- s\n- /\n- D\n- a\n- t\n- a\n- s\n- h\n- e\n- e\n- t\n- s\n- /\n- D\n- M\n- C\n- 3\n- 0\n- 7\n- 1\n- L\n- V\n- T\n- .\n- p\n- d\n- f\n\nGenerated with librepcb-parts-generator (generate_mosfet_dual.py)")
```

Comment:
Strange datasheet URL :grin:

Reply:
Oh I'm such an idiot... :joy: :man_facepalming:

Reply:
Should be fixed.